Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG APP_ROOT=/opt/app-root/src
-ARG BASE_IMAGE=docker.io/node:20.17.0-alpine
+ARG BASE_IMAGE=docker.io/node:22.12.0-alpine
```

Comment:
> Update node image to 20.18.1-alpine

This doesn't match the PR description. Are we bumping to v22 LTS as a part of this update? If we are doing the upgrade, `@tsconfig/node22` needs to be added in lieu of the node20 variant, and `@types/node` should also correlate.

Reply:
Fixed PR descroption typo. updated to `@tsconfig/node22`. 
Unfortunately `@types/node` is still on v20 variant so that will need to be updated at a later time.